src/main/java/com/sprint/mission/discodeit/service/BinaryContentService.java

```diff
@@ -0,0 +1,9 @@
+package com.sprint.mission.discodeit.service;
+
+import java.util.UUID;
+
+public interface BinaryContentService {
+    public void registerProfile(UUID userId, String filename);
+
+    public void deleteProfile(UUID userId);
```

Comment:
인터페이스는 접근제어자가 기본적으로 모두 public이라서 접근제어자를 선언할 필요 없습니다!
```suggestion
    void registerProfile(UUID userId, String filename);

    void deleteProfile(UUID userId);
```